OpenQA/Isotovideo/Backend.pm

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018-2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package OpenQA::Isotovideo::Backend;
+use Mojo::Base -base, -signatures;
+
+use Mojo::File qw(path);
+use backend::driver;
+use bmwqemu;
+
+sub new ($class) {
+    $bmwqemu::vars{BACKEND} ||= "qemu";
+    $bmwqemu::backend = backend::driver->new($bmwqemu::vars{BACKEND});
+
+    path('os-autoinst.pid')->spurt("$$");
+    # might throw an exception
+    $bmwqemu::backend->start_vm;
+}
+
+has process => sub {
+    return $bmwqemu::backend->{backend_process};
+};
+
+sub stop ($self) {
```

Comment:
Since the process might be `undef`:
```suggestion
sub stop ($self) {
    return undef unless my $process = $self->process;
```
And then you can use `$process` directly (instead of using the attribute repeatedly).